transformers/storage/vat/repository_test.go

```diff
@@ -781,6 +781,58 @@ var _ = Describe("Vat storage repository", func() {
 					Expect(queryErr).NotTo(HaveOccurred())
 					Expect(len(urnStates)).To(Equal(0))
 				})
+
+				It("handles removing snapshots when there's a three-part reorg", func() {
+					// this test case mimics the issue that is detailed in VDB-1756 where we saw a reorg that resulted in
+					// receiving three headers - canonical, noncanonical, canonical
+
+					// set up the header, diff and snapshot before the scenario being tested
```

Comment:
Why do you need all the "before" stuff?

Reply:
That's a good question - I probably don't need the before stuff, but I was adding it in there to more closely model what happened in prod. I wanted to prove that this trigger wouldn't manipulate data that was already in the database before the scenario in question, if that makes sense. Probably would be more clear if I was asserting against the art/ink values in the "before" snapshot to make sure that it remains unchanged. 

I'm also happy to remove all that before stuff, if it adds more confusion than is worth is.